lib/codecept.js

```diff
@@ -90,7 +90,7 @@ class Codecept {
   run(test) {
     this.mocha.files = this.testFiles;
     if (test) {
-      this.mocha.files = this.mocha.files.filter((t) => fsPath.basename(t, '_test.js') === test || fsPath.basename(t, '.js') === test || fsPath.basename(t) === test);
+      this.mocha.files = this.mocha.files.filter((t) => fsPath.basename(t).includes(test));
```

Comment:
but... `includes`? Can you provide something better for a filter? Maybe regex or glob patterns. Because that makes test execution unpredictable.